virtualBridge.py

```diff
@@ -70,6 +74,56 @@ def setup_dhcp_entry(self, cfg: NodeConfig) -> None:
         cmd = f"virsh net-update default add ip-dhcp-host \"{host_xml}\" --live --config"
         self.hostconn.run_or_die(cmd)
 
+    def remove_dhcp_entries(self, vms: list[NodeConfig]) -> None:
+        def filter_dhcp_leases(j: list[dict[str, str]], removed_macs: list[str], names: list[str]) -> list[dict[str, str]]:
+            filtered = []
+            for entry in j:
+                if entry["mac-address"] in removed_macs:
+                    logger.info(f'Removed host with mac {entry["mac-address"]}')
+                    continue
+                if "hostname" in entry and entry["hostname"] in names:
+                    logger.info(f'Removed host with name {entry["hostname"]}')
```

Comment:
there are several info logs here, that don't seem particularly interesting. They will fill up the terminal output. I think debug suffices.

Reply:
Note that if [th/log-once] branch would be merged, then you could always find all debug logs in `/tmp/cda-*log` file.

So if you run with info log, you usually don't need all the details there. And if you later need to debug, you still can find all logs afterwards.

If that would be accepted, then the info log should be really targeted for a nice and concise terminal output.